dpnp/dpnp_iface.py

```diff
@@ -757,6 +758,37 @@ def get_usm_ndarray_or_scalar(a):
     return a if dpnp.isscalar(a) else get_usm_ndarray(a)
 
 
+def is_cuda_backend(obj=None):
+    """
+    Checks that object has a cuda backend.
+
+    Parameters
+    ----------
+    obj : {Device, SyclDevice, SyclQueue, dpnp.ndarray, usm_ndarray, None},
+          optional
+        An input object with sycl_device property to check device backend.
+        If obj is  ``None``, device backend will be checked for the default
+        queue.
+        Default: ``None``.
+
+    Returns
+    -------
+    out : bool
+        Return ``True`` if object has a cuda backend, otherwise``False``.
+
+    """
+
+    if obj is None:
+        sycl_device = dpctl.SyclQueue().sycl_device
+    elif isinstance(obj, dpctl.SyclDevice):
+        sycl_device = obj
+    else:
+        sycl_device = getattr(obj, "sycl_device", None)
+    if sycl_device is not None and "cuda" in sycl_device.backend.name:
```

Comment:
```suggestion
    if sycl_device is not None and sycl_device.backend == dpctl.backend_type.cuda:
```
Mostly a trivial nit. Will probably be slightly faster, using an integer enumerator comparison rather than checking for sub-string membership.